### DISCUSSION TITLE
training losses aren't decreasing well

### DISCUSSION THREAD

**Discussion by mahootiha-maryam** [Project-MONAI/MONAI#4777] [General]
Hi, I wanted to use Dice loss for my segmentation. My images are kidneys with 1 channel. I wanted to use one_hot = True in front of dice loss and use sigmoid because my losses decrease very bad for example every 50 epochs it comes down 0.01 (training loss changes from 0.95 to 0.94 after 50 epochs). Do you know what could be the problem? I have to add I don't face error but I get this warning when I am training my model. 

```
UserWarning: single channel prediction, `to_onehot_y=True` ignored.
  warnings.warn("single channel prediction, `to_onehot_y=True` ignored.")
```

**Comment by KumoLiu** [comment]
Hi @mahootiha-maryam,
It seems your output only has 1 channel, if you want to use one_hot_y=True, you may need to train your model with output channel as 2.
Hope it will helps you!
